src/Components/Hero/Hero.scss

```diff
@@ -0,0 +1,65 @@
+﻿$curiosityThemeColor: grey;
+$spiritThemeColor: grey;
+$opportunityThemeColor: grey;
+
+
+.hero {
+    //temp
```

Comment:
instead of temp, a common pattern is to use `// TODO - can be removed once xxx`.

Most IDE's understand the word 'TODO' and make them really easy to spot / find across the whole project.
Its also a good idea to note down what it's blocked on, so that it can confidently be removed once that thing is done!